qiskit/aqua/operators/expectation_values/aer_pauli_expectation.py

```diff
@@ -0,0 +1,131 @@
+        self._state = state
+        self._snapshot_op = None
+
+    @property
+    def quantum_instance(self) -> QuantumInstance:
+        """ returns quantum instance """
+        return self._circuit_sampler.quantum_instance
+
+    @quantum_instance.setter
+    def quantum_instance(self, quantum_instance: QuantumInstance) -> None:
+        self._circuit_sampler.quantum_instance = quantum_instance
+
+    def expectation_op(self) -> OperatorBase:
+        """ expectation op """
+        # pylint: disable=import-outside-toplevel
+        from qiskit.providers.aer.extensions import SnapshotExpectationValue
+
+        # Construct snapshot op
+        # pylint: disable=inconsistent-return-statements
+        def replace_pauli_sums(operator):
+            if isinstance(operator, SummedOp):
+                paulis = [[meas.coeff, meas.primitive] for meas in operator.oplist]
+                snapshot_instruction = SnapshotExpectationValue('expval', paulis, variance=True)
+                snapshot_op = CircuitStateFn(snapshot_instruction, is_measurement=True)
+                return snapshot_op
+            if isinstance(operator, PauliOp):
+                paulis = [[operator.coeff, operator.primitive]]
+                snapshot_instruction = SnapshotExpectationValue('expval', paulis, variance=True)
+                snapshot_op = CircuitStateFn(snapshot_instruction, is_measurement=True)
+                return snapshot_op
+            if isinstance(operator, ListOp):
+                return operator.traverse(replace_pauli_sums)
+
+        snapshot_meas = replace_pauli_sums(self._operator)
+        return snapshot_meas
+
+    def compute_expectation(self,
+                            state: OperatorBase = None,
+                            params: dict = None) -> Union[float, complex, OperatorBase]:
+        # Wipes caches in setter
+        if state and not state == self.state:
+            self.state = state
+
+        if 'QuantumCircuit' in self.state.get_primitives():
+            if not self._snapshot_op:
+                snapshot_meas = self.expectation_op()
+                self._snapshot_op = snapshot_meas.compose(self.state).reduce()
+
+            measured_op = self._circuit_sampler.convert(self._snapshot_op, params=params)
+            # TODO once https://github.com/Qiskit/qiskit-aer/pull/485 goes through
+            # self._quantum_instance._run_config.parameterizations = ...
+            # result = self.quantum_instance.execute(list(self._snapshot_circuit.values()))
+            return measured_op.eval()
+        else:
+            # If no circuits to run (i.e. state is a Dict, eval directly)
+            return StateFn(self._operator, is_measurement=True).eval(self.state)
+
+    def compute_standard_deviation(self,
+                                   state: OperatorBase = None,
+                                   params: dict = None) -> float:
+        """ compute standard deviation """
+        return 0.0
```

Comment:
Should this be 0? Or did I not understand the code correctly and only statevector simulation is used?

Reply:
Yup, only statevector is used.